tasks/main.yml

```diff
@@ -34,10 +34,10 @@
   environment:
     NPM_CONFIG_PREFIX: "{{ npm_config_prefix }}"
     NODE_PATH: "{{ npm_config_prefix }}/lib/node_modules"
-    NPM_CONFIG_UNSAFE_PERM: "{{ npm_config_unsafe_perm }}"
+    NPM_CONFIG_UNSAFE_PERM: "{{ npm_config_unsafe_perm | bool }}"
   with_items: "{{ nodejs_npm_global_packages }}"
 
 - name: Install packages defined in a given package.json.
   npm:
     path: "{{ nodejs_package_json_path }}"
-  when: nodejs_package_json_path is defined and nodejs_package_json_path
+  when: nodejs_package_json_path is defined and nodejs_package_json_path | bool
```

Comment:
This should already be a bool too, I think?

Reply:
It might be but doesn't have to, that's why we're coercing it

Reply:
Doesn't `and` make it a bool?

Reply:
I added parentheses to make the intention here clearer